mos_tests/neutron/python_tests/test_dhcp_agent.py

```diff
@@ -56,25 +56,7 @@ def prepare_openstack(self, setup):
     def isclose(self, a, b, rel_tol=1e-9, abs_tol=0.0):
         return abs(a - b) <= max(rel_tol * max(abs(a), abs(b)), abs_tol)
 
-    @pytest.mark.testrail_id('542614')
-    def test_to_check_dhcp_agents_work(self):
-        """[Neutron VLAN and VXLAN] Check dhcp-agents work
-
-        Steps:
-            1. Update quotas for creation a lot of networks:
-                neutron quota-update --network 1000 --subnet 1000
-                                     --router 1000 --port 1000:
-            2. Create 50 networks, subnets, launch and terminate instance
-            3. Get the table with all agents:
-                neutron agent-list
-            4. Checl networks on each dhcp-agent:
-                neutron net-list-on-dhcp-agent <id_agent_from_the_table>
-                Check that there are nets on all of agent
-            5. Check networks quantity on each dhcp-agent:
-                neutron net-list-on-dhcp-agent <id_agent_from_the_table>
-              Check that quantity on agents are nearly equal
-        """
-
+    def create_50_networks(self, ):
```

Comment:
stange method name, usually number are not welcomed in func names, but numbers provides as func arguments, for ex.:

```python
def create_networks(self, number):

self.create_networks(50)
```